src/typescript/frontend/src/components/pages/emojicoin/components/main-info/MainInfo.tsx

```diff
@@ -71,10 +81,12 @@ const MainInfo = ({ data }: MainInfoProps) => {
               }
         }
       >
-        <Emoji
+        <Link
           className="text-[24px] text-center md:display-2 my-auto text-white"
-          emojis={data.emojis}
-        />
+          href={explorerLink}
+        >
+          <Emoji emojis={data.emojis} />
+        </Link>
```

Comment:
This link is great, is there also a way to make a link show up just below the clickable emoji too? Mainly because people might not realize immediately that it is clickable or copy-pastable

For example on https://emojicoin-dot-15qssceh7-econia-labs.vercel.app/market/laptop;high_voltage, just below the emoji at the top that is now clickable, there would be a link right below that looks something like:

[`0x43dc...eb67`](https://explorer.aptoslabs.com/coin/0x43dcf02dcc0f3759d00486052585bf1694acf85c7e3e7c4b4770c5216d58eb67::coin_factory::Emojicoin?network=testnet) [the window icon that indicates copy-paste]

Basically have it look as much as possible like the explorer

- [ ] Verify on mobile
- [ ] Verify on desktop
- [ ] Verify on other screens sizes are required